### PR TITLE
UIの変更

### PR DESCRIPTION
**Pull request by kugimasa**
# 変更点
- `index.html`, `p2p-media.html`共にアスペクト比をレスポンシブに対応しました。
- 招待コードをp2p通信の準備ができた時のみコピーするようにしました。
- 招待コードを表示せずにコピーできるようにしました。

### CODE REVIEW

**Comment by kugimasa** [comment]
**hosting**はまだしていません🙇‍♂️

File: public/assets/js/invite.js [hunk 1]
```diff
@@ -1,12 +1,15 @@
 var invite = function() {
     var url = location.href;
     const localId = document.getElementById('js-local-id').innerText;
-
-    var inviteUrl = document.getElementById('js-invite-url');
+    if (localId == "No User"){
+        return;
+    }
```
Comment (daiking1756) [inline public/assets/js/invite.js]: Fantastic! I didn't realize this method.

IMO(in my opinion), from the perspective of UX, before return, we should notify to user that the copy is failed.
The simplest case is like bellow code.
```case1
if (localId == "No User"){
    UIkit.notification({
        message: 'コピー失敗、IDが表示されてからもう一度招待ボタンを押して下さい',
        status: 'dan ger',
        pos: 'top-center',
        timeout: 2000
    });
    return;
}
```
Other case
1. first define `message` val (I think we can also initialize this val here, not only define)
2. just modify message `if localId==NoUser`
3. goto UIKit.notification() (it meens, skip process about copy the invite url to clipboard)

```case2
var message = 'コピー失敗、IDが表示されてからもう一度招待ボタンを押して下さい';
if (localId == "No User"){
    // goto UIKit.notification
}

```
IMO, Ideally we should modify the specification of invite button not to be able to click before available the SkyWay peer ID by adding 'disable' property, that is HTML5 power, to our invite button.

Could you try modifying your code to improve UX?

**Review by daiking1756 — CHANGES_REQUESTED** [review]
Reviewed!
Fantastic code!
But. I think you can improve your code yet.
You can do it.
I wrote comment about detail of my opinion.
Please refer to...

File: public/assets/js/invite.js (hunk 1 above)
Comment (kugimasa) [reply public/assets/js/invite.js]: Thanks for the review!!
I  totally agree about giving the notification when copy fails.
I'll fix it 👍

**Comment by kugimasa** [comment]
@daiking1756 
I have added a commit to disable the invite button when `localId` is not loaded.
Due to this I didn't add the feed back notification.
(Since there is no chance of pushing the button when the id isn't loaded 😉  )

**Pull request merged**